README.md

```diff
@@ -1,27 +1,49 @@
-# epm
-
-Ethereum Package Managment
+# EPM
 
 [![Join the chat at https://gitter.im/ethpm/Lobby](https://badges.gitter.im/ethpm/Lobby.svg)](https://gitter.im/ethpm/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-# Goals
+Ethereum Packaging Specification
```

Comment:
A bit more verbose but it could also be *"Ethereum Smart Contract Packaging Specification"* which may have more meaning to more people and still convey the same intent.  Or *"EVM Smart Contract Packaging Specification"*